hooks/useFeed.tsx

```diff
@@ -8,29 +8,34 @@ const useFeed = ({ feedType }: { feedType: string }) => {
   const { filter } = useFeedProvider();
 
   const fetchPoints = async (start: number) => {
-    const query = supabaseClient.from('posts').select('*').not('likes', 'is', null);
+    if (feedType == FeedType.Recent) {
+      const query = supabaseClient.from('posts').select('*').not('likes', 'is', null);
 
-    if (filter?.platform) {
-      query.eq('platform', filter.platform);
-    }
+      if (filter?.platform) {
+        query.eq('platform', filter.platform);
+      }
 
-    if (filter?.channel) {
-      query.like('parent_url', `%${filter.channel}%`);
-    }
+      if (filter?.channel) {
+        query.like('parent_url', `%${filter.channel}%`);
+      }
 
-    if (feedType == FeedType.Recent) {
       query.order('created_at', { ascending: false });
       query.range(start, start + 5);
-    } else {
-      query.order('likes', { ascending: false });
-      query.range(start, start + 20);
-    }
```

Comment:
why are we removing the difference in filter for
- trending vs
- recent?

2 different feed types right?
- recent: needs most recent posts
- trending: needs most liked posts

your updates will only get `trending` posts sorted in `recent` ordering right?